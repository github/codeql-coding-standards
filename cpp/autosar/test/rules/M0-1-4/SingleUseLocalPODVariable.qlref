rules/M0-1-4/SingleUseLocalPODVariable.ql