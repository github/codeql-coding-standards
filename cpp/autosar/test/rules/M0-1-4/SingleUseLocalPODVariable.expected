| test.cpp:12:5:12:5 | b | Local POD variable b in test_unused_local is only $@. | test.cpp:13:3:13:3 | b | used once |
| test.cpp:14:7:14:7 | i | Local POD variable i in test_unused_local is only $@. | test.cpp:14:10:14:11 | initializer for i | used once |
| test.cpp:25:5:25:5 | t | Local POD variable t in f1 is only $@. | test.cpp:26:3:26:3 | t | used once |
